### PR TITLE
add human skeleton rviz visualization

### PR DESCRIPTION
**Pull request by knorth55**
waiting for upstream 
depend on https://github.com/jsk-ros-pkg/jsk_recognition/pull/2437

This is visualization for https://github.com/jsk-ros-pkg/jsk_recognition/pull/2437.

- add human skeleton visualization for `HumanSkeletonArray` msg

human_skelton
human_skelton_2

### CODE REVIEW

File: jsk_rviz_plugins/src/human_skeleton_array_display.h [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+#endif
+
+namespace jsk_rviz_plugins
+{
+  class HumanSkeletonArrayDisplay:
+    public rviz::MessageFilterDisplay<jsk_recognition_msgs::HumanSkeletonArray>
+  {
+    Q_OBJECT
+  public:
+    typedef std::shared_ptr<rviz::BillboardLine> BillboardLinePtr;
```
Comment (YutoUchimi) [inline jsk_rviz_plugins/src/human_skeleton_array_display.h]: Please support indigo like https://github.com/jsk-ros-pkg/jsk_visualization/pull/739 .

Comment (knorth55) [reply jsk_rviz_plugins/src/human_skeleton_array_display.h]: this PR doen't support Indigo and current branch support only kinetic and melodic.
This PR should be merged first and solve this indigo problem in #739 later.

Comment (YutoUchimi) [reply jsk_rviz_plugins/src/human_skeleton_array_display.h]: OK. Then, looks good to me.

**Comment by knorth55** [comment]
`jsk_recognition` `1.2.15` is released.
So now this PR is no more waiting upstream.

**Pull request closed without merging**